### PR TITLE
Feat/cloud labels

### PR DESCRIPTION
**Pull request by MiroslavRepka**
This PR implements #352. While doing so, I also formatted/unified the templates to be more readable as templates and terraform files after generation. 

Other chores:
- Update of the provider versions (only minor version changed)
- Small bugfix in the `command` package

All of the templates were tested locally to ensure they work as expected.

### CODE REVIEW

File: services/terraformer/templates/gcp-lb.tpl [hunk 1]
```diff
@@ -50,39 +51,45 @@ resource "google_compute_firewall" "firewall-{{ $region }}" {
...
+    subnetwork = google_compute_subnetwork.{{ $nodepool.Name }}_subnet.self_link
     access_config {}
   }
   metadata = {
     ssh-keys = "root:${file("./public.pem")}"
   }
   metadata_startup_script = "echo 'PermitRootLogin without-password' >> /etc/ssh/sshd_config && echo 'PubkeyAuthentication yes' >> /etc/ssh/sshd_config && service sshd restart"
+
+  labels = {
+    environment = "claudie"
```
Comment (Despire) [inline services/terraformer/templates/gcp-lb.tpl:86]: "Managed by claudie" wouldn't work here ?

**Review by Despire — APPROVED** [review]
LGTM, thanks @MiroslavRepka

**Review by MarioUhrik — COMMENTED** [review]
Hi Miro, nice work

I think we should leave the `environment` tag unused.
It's probably the most commonly used tag, people may want to control it themselves.
Instead, something like `managedBy: claudie` may work better.

Also, how about project/cluster specific cloud tags?
e.g. having both:
- `managedBy: claudie`
-  `claudieCluster: $CLUSTERNAME`
-  `claudieProject: $PROJECTNAME`

File: services/terraformer/templates/gcp-lb.tpl (hunk 1 above)
Comment (MiroslavRepka) [reply services/terraformer/templates/gcp-lb.tpl:86]: The GCP labels have a strict requirements, like no capitals, no white spaces. We can do `managed-by-claudie` though.

**Comment by MiroslavRepka** [comment]
Hi @MarioUhrik, thanks for the feedback. I added `cluster name` to the tags/labels/descriptions, however, I did not add `project name`, as that would require some changes in the code. If you think we want to have it there, it can be done. :slightly_smiling_face:

**Review by MarioUhrik — APPROVED** [review]
Thanks Miro, I'm happy with this :)

**Pull request merged**